add antiscroll@0.9 w/ git auto-update

nolsherry opened the pull request:
PR for #8434 
@pvnr0082t , @kennynaoh , please help me review this PR, thanks!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/Automattic/antiscroll/
- Official website (optional, not the repository): https://automattic.github.io/antiscroll/
- NPM package url (optional):
- GitHub / Bitbucket popularity (required):
  - Count of stars: 1039
  - Count of watchers: 216
  - Count of forks: 171
- NPM download stats (optional):
  - Downloads in the last day:
  - Downloads in the last week:
  - Downloads in the last month:
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [X] This lib was not found on cdnjs repo
- [X] No already exist / duplicated issue and PR
- [X] The lib has notable popularity
  - [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [ ] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [X] Has valid tags for each versions (for git auto-update)
- [X] Auto-update setup
- [X] Auto-update target/source is valid.
- [X] Auto-update filemap is correct.
# Git commit checklist
- [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [X] The parent of the commit(s) in the PR is not old than 3 days.
- [X] Pull request is sending from a non-master branch with meaningful name.
- [X] Separate unrelated changes into different commits.
- [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [X] Close corresponding issue in commit message
- [X] Mention related issue(s), people in commit message, comment.

close #8434, cc @nylen

pvnr0082t commented on ajax/libs/antiscroll/package.json:
```diff
@@ -0,0 +1,30 @@
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/nylen/antiscroll"
```
@nolsherry where did you get this url? This link shows `404 not found`.

nolsherry replied:
@pvnr0082t, sorry for making this mistake, I mistook the `url` in `repository` field, and I modified `url`, thanks for your help!

nolsherry commented:
@pvnr0082t , @kennynaoh , I have modified the wrong `url` into the correct `url` in `package.json`,
Please help me review this PR, thanks!

pvnr0082t commented:
LGTM

nolsherry commented:
@pvnr0082t , no problem, I will contribute `package.json` to upstream after this PR is merged successfully.

PeterDaveHello commented:
what about scrollbar.js?

nolsherry commented:
@PeterDaveHello, I will confirm with author whether the file is necessary.

nolsherry commented:
@PeterDaveHello the author does not response me for a while ([antiscroll/issues#97](https://github.com/Automattic/antiscroll/issues/97)), should we add `scrollbar.js` ?

PeterDaveHello commented:
Maybe we can just add that file.

PeterDaveHello commented:
Any updates here? Thanks!

nolsherry commented:
@PeterDaveHello sorry for the late reply, I have added `scrollbar.js`.
@kennynaoh I have modified `package.json`, and added  `scrollbar.js`,
please help me review this PR, thanks!
